### DISCUSSION TITLE
Shelly 1 v3 not working after update to 12.7/12.7.1

### DISCUSSION THREAD

**Discussion by pandizajner** [RavenSystem/esp-homekit-devices#2082] [MEPLHAA Scripts]
**Detailed steps to reproduce it**

I upgraded a device to HAA 12.7 (by accidental triggering as several time on/off switching)  and the device stopped working. I have Shelly 1 v3 working in 12V DC connected with LED bar and touch switch. I downloaded HAA Manager to investigate and then upgraded this device from the app to 12.7.1. There’s still the same problem: the device doesn’t react on manual switch as well as triggered from Home app. No “click” sound, no nothing. 

I just updated the other same model device from 10.X to 12.7.1 and exactly the same issue happened. No reaction on clicking and manual switching.

It’s worth to be addded: both devices are displayed in HAA Manager as “active” and “blocked” saying no. Rebooting and updating don’t provide any solution here.

**MEPLHAA Script**

```{"a":[{"0":{"r":[{"g":4}]},"1":{"r":[{"g":4,"v":1}]},"b":[{"g":5,"p":0},{"g":5,"t":0}],"s":0}]}```

**Used device**

2 x Shelly 1 v3

**HAA version**

Happened on 12.7 and 12.7.1

**iOS/iPadOS version**

16.5.1

**Logs**
Unfortunately I wasn’t flashing the device on my own, so it’s hard for me to investigate the device logs.

**Screen-shots or videos showing issue**

**Comment by pandizajner** [comment]
Oh, I see now there was some architecture update with v12 that requires some script migration. 
Is there any chance somebody could help me update the MEPLHAA script to be aligned with the new standards? I read the [guide](https://github.com/RavenSystem/esp-homekit-devices/wiki/Migration-Guide-for-HAA-V12-Merlin) but it's really hard for me to get the idea what and how I should change in it.

**Comment by RavenSystem** [comment]
All scripts from the wiki have been updated. You can use any of them. Standard Shelly 1 scripts are here:
https://github.com/RavenSystem/esp-homekit-devices/wiki/devices-database

If you use custom scripts with special functions, post them here to help you to update.

**Comment by pandizajner** [comment]
@RavenSystem Thank you! I just figured out the tableview is horizontally scrollable and the scripts are really there! That wasn't obvious the first time I got there.
I just found another (polish) [source](https://test.energo-dom.pl/kody-konfiguracji-json-dla-haa-homekit/#1682504053676-ab51c08a-2275) of scripts for Shelly and other devices – not sure though if that's not mimicing your content.
